packages/timeago_flutter/pubspec.yaml

```diff
@@ -7,12 +7,15 @@ environment:
   sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  timeago: ^3.3.0
```

Comment:
I would keep this change, and not add the manual dependencies override in your PR

Reply:
Oh, okay...